### DISCUSSION TITLE
物料列面的schema中的properties MetaSelect的options能获取接口数据吗？

### DISCUSSION THREAD

**Discussion by w990752800** [opentiny/tiny-engine#388] [Q&A]
![image](https://github.com/opentiny/tiny-engine/assets/23731304/a55406ab-15ed-4946-9483-bb0474c33348)

**Comment by w990752800** [comment]
我想拿服务器的数据

**Reply by chilingling** [reply]
引擎开源方案暂时还不支持配置式获取服务端数据，因为这里需要拿服务端数据的话，是业务方的数据，会涉及到鉴权、token 相关的问题。
目前您可以先手动修改支持，或者等待官方开源方案。
@w990752800
